enhancements/kueue/openshift-kueue.md

```diff
@@ -461,18 +534,48 @@ I don't forsee any issue with SNO.
 
 Cert Manager will be used to manage certificates so our operator will have a hard dependency on Cert Manager.
 
+#### Tech Preview Feature Statement
+
+##### Supported Features
+
+In Tech preview, we will provide the following features:
+
+- [Preemption](https://kueue.sigs.k8s.io/docs/concepts/preemption/)
+- Advanced Resource management: Comprising: resource flavor fungibility, fair sharing, cohorts and preemption with a variety of policies between different tenants.
+- Support for GPU model servers via pod-integration
+- Namespace opt in for frameworks.
+- Metrics for Kueue in OCP monitoring.
+- [Workload Priority Classes](https://kueue.sigs.k8s.io/docs/concepts/workload_priority_class/)
+- LocalQueue, ClusterQueues, Workloads, Worklow Priorites, Admission Checks are all supported.
+- Support for deployments, pods, statefulsets.
+- [Use of resource flavors to describe heterogeous clusters](https://kueue.sigs.k8s.io/docs/concepts/resource_flavor/)
+
+##### Not Supported
```

Comment:
Not even in 4.20 as TP still? Is there a timeline for any of these features? Would be good to distinguish those we intend to support over time vs not at all

Reply:
updated.